craft_parts/packages/deb.py

```diff
@@ -202,6 +209,18 @@
 }
 
 
+def _apt_cache_wrapper(method):
+    """Decorate a method to handle apt availability."""
+
+    @functools.wraps(method)
+    def wrapped(self, *args, **kwargs):
```

Comment:
Signature here does not correspond to the decorated methods (which are class methods, receive `cls`, not `self`) -- probably just leaving `*args, **kwargs` is good enough.